src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -84,15 +94,15 @@ def spring_cloud_create(cmd, client, resource_group, name, location=None, app_in
         if enable_java_agent:
             client_preview = get_mgmt_service_client(cmd.cli_ctx, AppPlatformManagementClient_20201101preview)
             logger.warning("Start configure Application Insights")
-            trace_properties = update_java_agent_config(cmd, resource_group, name, location, app_insights_key, app_insights,
-                                                        enable_java_agent)
+            trace_properties = update_java_agent_config_with_connection_string_as_default(cmd, resource_group, name, location, app_insights_key, app_insights,
```

Comment:
can we just rename the term from `trace` to `monitoringSettings` ?

Reply:
`trace_properties` is spreading all over the code, around 50 locations, adding risks to do a one-time replacement

Reply:
I see, we can file another PR for this.

Reply:
Sure, I'll create another work item to keep track of this